backend/pkg/api/api.go

```diff
@@ -173,7 +173,10 @@ func setDefaultClientProviders(cfg *config.Config, logger *zap.Logger, opts *opt
 
 // Start the API server and block
 func (api *API) Start() {
-	err := api.ConsoleSvc.Start()
+	startCtx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
Ideally I think this could be based on `cfg.Kafka.Startup` settings (ie doing the math to determine the max it could take)... with some absoulte maximum value . that is we take minimum of "calculated based on settings" and "default absolute max"